packages/inter-protocol/src/proposals/add-auction.js

```diff
@@ -143,6 +152,8 @@ export const addAuction = async ({
   );
   // don't overwrite auctioneerKit or auction instance yet. Wait until
   // upgrade-vault.js
+
+  auctionsDone.resolve(true);
```

Comment:
is there no other way to indicate this than this boolean in promise space?

if it must be in this somewhat global space please give it a more specific name. Maybe by a general pattern we can use to know a value is one-off. E.g. `didAuctionsOverUpgrade16`